updated Dashboard, Pivot EUD 20.1 (#107)
DevExpress/dotnet-eud: dashboard-for-desktop/articles/dashboard-designer/providing-data/binding-to-federated-data-source.md

   ![Choose_federated_data_source](../../../images/choose-federated-data-source.png)

> 3. The [Query Builder](../../dashboard-designer/working-with-data/using-the-query-builder.md) dialog displays the available data sources.

Reviewer: ```suggestion
3. The [Query Builder](../../dashboard-designer/working-with-data/using-the-query-builder.md) dialog displays available data sources.
```

   ![Query_Builder_available_sources](../../../images/query-builder-available-sources.png)

4. Select the **Query Type**:

   **Join**

>    Combines rows from two or more tables based on a column they share. The join type specifies records that have matching values in both tables.

Reviewer: ```suggestion
   Combines rows from two or more tables based on a shared column. The 'join' type specifies records with matching values in both tables.
```

>     Drag-and-drop the data sources, specify the related column to create the relationship between tables, and select the columns you want to include in the query. Columns included in the query are displayed in the bottom pane where you can configure their settings.

Reviewer: ```suggestion
    Drag-and-drop the required data sources, specify the related column to create a relationship between tables, and select columns to include in the query. Columns included in the query are displayed in the bottom pane - where you can configure their settings.
```

   ![Federated_data_source_settings](../../../images/federated-source-settings.png)

   **Union**

>    Combines rows from two or more tables into a single data set and removes duplicate rows in merged tables. You can only create a union query for data sources that contain similar columns with the same names. Data types of such columns should be [implicitly converted](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/types/casting-and-type-conversions#implicit-conversions).

Reviewer: ```suggestion
   Combines rows from two or more tables into a single data set and removes duplicate rows in merged tables. You can only create a union query for data sources that contain similar columns with the same name. Data types of such columns should be [implicitly converted](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/types/casting-and-type-conversions#implicit-conversions).
```

>    Drag-and-drop the data sources to combine them in one. Unlike the join type, you cannot specify which columns to include in the union query. Columns included in the query are displayed in the bottom pane where you can specify the aliases of the newly created columns.

Reviewer: For my understanding, are multiple data sources being combined into a single data source?

Author: In general - yes, https://www.essentialsql.com/what-is-the-difference-between-a-join-and-a-union/

Reviewer: ```suggestion
   Drag-and-drop the data sources to combine them into a single data source. Unlike a join query, you cannot specify columns to include in a union query. Columns included in the query are displayed in the bottom pane, where you can specify the aliases of the newly created columns.
```

   ![](../../../images/data-federation-querybuilder-union.png)

   **Union All** 

   Operates in the same manner as **Union**, but duplicates rows from different tables when they contain the same data.

   ![](../../../images/data-federation-querybuilder-union-all.png)

   When you are finished, click **OK**.
 

> 5. The [Data Source Browser](../ui-elements/data-source-browser.md) displays the newly created Federation Data Source.

Reviewer: What is a "Federation Data Source"? Is this the correct technical term?

Author: ```suggestion
5. The [Data Source Browser](../ui-elements/data-source-browser.md) displays the newly created Federated Data Source.
```

Reviewer: Thank you. Please apply your suggestion.
